components/service/glean/docs/pings.md

```diff
@@ -0,0 +1,42 @@
+# glean pings
+
+glean provides a set of pings that are assembled, out of the box, without any developer intervention.
+Every glean ping is in JSON format and contains a [common section](#the-common-information-section)
+with shared information data, included under the `ping_info` key.
+The following is a list of the pings, along with the conditions that triggers them.
+
+- [`baseline` ping](baseline.md)
+- `events` ping (TBD)
+- `metrics` ping (TBD)
+
+## The `common` information section
+The following fields are always included in this section, for every ping.
+
```

Comment:
Do we want a section detailing when/how these are generated, cleared, and regenerated? For instance, I thought I remembered that there was a profile reset or opt-out mechanism that reset the clientId or reset  `seq` to 0 or something...

Reply:
Good point. We don't have, right now, anything to reset user-level metrics. They go away when user uninstalls the product using glean. I filed bug 1513616 in case some products want to have a way to clear stuff manually without uninstalling the app.